convert2rhel/checks.py

```diff
@@ -295,6 +296,17 @@ def ensure_compatibility_of_kmods():
         logger.info("Kernel modules are compatible.")
 
 
+def validate_yum_transaction():
+    """ """
+    logger.info("\nValidate the %s transaction", pkgmanager.TYPE)
```

Comment:
Method says validate yum, but then the `pkgmanager.TYPE` indicates either yum or dnf

Reply:
I was using `yum` here in this context to be a generic way of saying {yum,dnf}, but I guess it is confusing to someone else that will read this. I will change it.